streamer/src/nonblocking/stream_throttle.rs

```diff
@@ -12,14 +11,16 @@ use {
     tokio::time::sleep,
 };
 
-const MAX_UNSTAKED_STREAMS_PERCENT: u64 = 20;
 pub const STREAM_THROTTLING_INTERVAL_MS: u64 = 100;
 pub const STREAM_THROTTLING_INTERVAL: Duration =
     Duration::from_millis(STREAM_THROTTLING_INTERVAL_MS);
 const STREAM_LOAD_EMA_INTERVAL_MS: u64 = 5;
 const STREAM_LOAD_EMA_INTERVAL_COUNT: u64 = 10;
 const EMA_WINDOW_MS: u64 = STREAM_LOAD_EMA_INTERVAL_MS * STREAM_LOAD_EMA_INTERVAL_COUNT;
 
+/// Maximum TPS for unstaked connections
```

Comment:
nit: this comment still makes it a little ambiguous for the casual observer if this TPS limit is per connection or in aggregate across all unstaked connections

Reply:
I'm reworking SWQOS anyway, will address this also.